phylogenetic/rules/export.smk

```diff
@@ -25,6 +25,22 @@ This part of the workflow usually includes the following steps:
 See Augur's usage docs for these commands for more details.
 """
 
+rule colors:
+    input:
+        color_schemes = "defaults/color_schemes.tsv",
+        color_orderings = "defaults/color_orderings.tsv",
+        metadata = "data/metadata_{segment}.tsv",
+    output:
+        colors = "results/colors_{segment}.tsv"
+    shell:
+        """
+        python3 scripts/assign-colors.py \
```

Comment:
_non-blocking_
Another reminder for us that there's an open issue to add this functionality to augur: https://github.com/nextstrain/augur/issues/1185